kahuna/public/js/edits/image-editor.html

```diff
@@ -80,6 +83,15 @@
                                 image="ctrl.image">
             </gr-archiver-status>
         </div>
+
+        <div ng-if="ctrl.isDeleted" class="job-status status deleted_image_warning"
+             ng-class="{
+                         'status--invalid': !ctrl.canUndelete
+                     }">
+            <span ng-if="ctrl.canUndelete">Image deleted (this image already exist and has been deleted. Click Add to Library to undelete it)</span>
+            <span ng-if="!ctrl.canUndelete">Image deleted (this image already exist and has been deleted. Click View image to see details)</span>
```

Comment:
```suggestion
            <span ng-if="!ctrl.canUndelete">Image deleted (this image already exists and has been deleted. Click View image to see details)</span>
```

Sorry, my typos in mockups.